arex-instrumentation/config/arex-apollo/src/main/java/io/arex/inst/config/apollo/ApolloServletRequestHandler.java

```diff
@@ -0,0 +1,61 @@
+package io.arex.inst.config.apollo;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import com.google.auto.service.AutoService;
+import io.arex.agent.bootstrap.model.MockCategoryType;
+import io.arex.agent.bootstrap.util.StringUtil;
+import io.arex.inst.runtime.context.ContextManager;
+import io.arex.inst.runtime.model.ArexConstants;
+import io.arex.inst.runtime.request.RequestHandler;
+
+@AutoService(RequestHandler.class)
+public class ApolloServletRequestHandler implements RequestHandler<HttpServletRequest, HttpServletResponse> {
+    @Override
+    public String name() {
+        return MockCategoryType.SERVLET.getName();
```

Comment:
Is there a need to differentiate between V3, V5

Reply:
V5 init may ClassNoDef exception, i will refactor RequestHandler on next branch